src/test/features/claimant-response/filters/frequency-view-filter.ts

```diff
@@ -17,6 +18,26 @@ describe('Frequency view filter', () => {
   })
 
   context('render payment frequency', () => {
-    // TODO: https://tools.hmcts.net/jira/browse/ROC-5325
+
+    for (let paymentFrequency in PaymentFrequency) {
+      Frequency.all()
+        .forEach(frequency => {
+          if (Frequency.of(paymentFrequency).inWeeks === frequency.inWeeks) {
+            it(`should render payment frequency ${paymentFrequency} as ${frequency.displayValue}`, () => {
+              expect(FrequencyViewFilter.renderPaymentFrequency(
+                Frequency.toPaymentFrequency(Frequency.of(paymentFrequency)))).to.equal(frequency.displayValue)
+            })
+          } else {
+            it(`should not render payment frequency ${paymentFrequency} as ${frequency.displayValue}`, () => {
```

Comment:
I don't think we need to test these inequalities; it's just complicating things.